src/community/security/oauth2-openid-connect/src/main/java/org/geoserver/security/oauth2/OpenIdConnectAuthenticationFilter.java

```diff
@@ -69,17 +68,26 @@ protected Collection<GeoServerRole> getRoles(HttpServletRequest request, String
         return super.getRoles(request, principal);
     }
 
+    public static Object extractFromJSON(String json, String path) {
```

Comment:
```suggestion
    static Object extractFromJSON(String json, String path) {
```

Reply:
Reduce visibility (suggested change) or document public method with javadoc (surprised a code QA check has not caught this?).